lib/decision_review/schemas.rb

```diff
@@ -1,63 +1,65 @@
 # frozen_string_literal: true
 
-module DecisionReview::Schemas
-  nod_create_request_schema_json_string = File.read(
-    Rails.root.join('lib', 'decision_review', 'schemas', 'NOD_create_request_body_schema.json')
-  )
+module DecisionReview
+  module Schemas
```

Comment:
going from `::` to 2 lines is the only change, this is a really unfortunate diff

Reply:
"Hide whitespace changes" under the cog.

Reply:
😮